packages/core-data/src/actions.js

```diff
@@ -209,29 +214,73 @@ export function* redo() {
  * @param {string} kind    Kind of the received entity.
  * @param {string} name    Name of the received entity.
  * @param {Object} record  Record to be saved.
+ * @param {Object} options Saving options.
  */
-export function* saveEntityRecord( kind, name, record ) {
+export function* saveEntityRecord(
+	kind,
+	name,
+	record,
+	{ isAutosave = false } = { isAutosave: false }
```

Comment:
Based on the code changes, it feels that the common code is very small between autosaves and saves, which makes me wonder if they should be two separate action creators.

Reply:
I thought that maybe future `options` could make them share more code.

Should we separate them for now?

Reply:
I'm fine either way, you're in a better position to judge as you worked on it :)